### ISSUE TITLE
Create ability to delete email addresses created in error

### ISSUE THREAD

**Issue by jjchen20** [razzius/hms-weave#141]
There's an error when trying to remove an email account with a typo: 
![image](https://user-images.githubusercontent.com/43801169/69475029-f40fbf80-0d7c-11ea-9052-414ef65f2a0e.png)

Is there a way to manually remove incorrect/unwanted email accounts?

**Comment by razzius** [comment]
It's possible to delete the Verification Email if you delete all the Verification Tokens first. That's a hassle with the current admin UI, however, so I'll look into making this smoother.

**Comment by jjchen20** [comment]
It's helpful to see how many people log in by keeping the tokens there. We
also don't want to interfere with other people's tokens and sessions within
a two week time period.

**Comment by razzius** [comment]
Right, I should clarify: you have to delete all Verification Tokens relating to that Verification Email. Other emails' tokens don't need to be involved.

**Comment by jjchen20** [comment]
That would be fine if it's related just to that one user. I would only be
deleting an email because it's an invalid email address.
